cmd/apikey_save.go

```diff
@@ -1,26 +1,51 @@
 package cmd
 
 import (
+	"bufio"
 	"fmt"
 	"os"
 
 	"github.com/civo/civogo"
 	"github.com/civo/cli/config"
 	"github.com/civo/cli/utility"
 	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
 var apikeySaveCmd = &cobra.Command{
 	Use:     "save",
 	Aliases: []string{"add", "store", "create", "new"},
 	Short:   "Save a new API key",
-	Args:    cobra.MinimumNArgs(2),
+	Args:    cobra.MinimumNArgs(0),
 	Example: "civo apikey save NAME KEY",
 	Run: func(cmd *cobra.Command, args []string) {
-		config.Current.APIKeys[args[0]] = args[1]
 
+		var username, apiKey string
```

Comment:
not `username`, just `name`. Remember, it's not their username but a key.

`civo apikey save dev hodihsofidjsoifjdsoijf`

You can use whatever you want as the name, it's just a reference for when listing them in the CLI, it isn't used anywhere.